clients/include/testing_gemm_batched_ex.hpp

```diff
@@ -391,8 +369,15 @@ void testing_gemm_batched_ex(const Arguments& arg)
         else
             rocblas_init<To>(hC[b], M, N, ldc);
         rocblas_init<To>(hD_1[b], M, N, ldd);
-        hD_2[b]    = hD_1[b];
-        hD_gold[b] = hD_1[b];
+    }
+
+    hD_2.copy_from(hD_1);
+    for(int b = 0; b < batch_count; b++)
+    {
+        for(int i = 0; i < size_d; i++)
```

Comment:
size_t i

Reply:
> 
> 
> size_t i

I have an upcoming PR on this issue, a source-wide change to use integer types better.